tests/fixtures/name-matches-to-wikidata.json

```diff
@@ -0,0 +1,96 @@
+{
+  "compareFunction": "name-matches-to-wikidata",
+  "fixtures": [
+    {
+      "description": "Test OSM name different from Wikidata name",
+      "expectedResult": {
+        "result:name_matches_to_wikidata": false
+      },
+      "newVersion": {
+        "type": "Feature",
+        "properties": {
+          "osm:id": 427818536,
+          "osm:type": "way",
+          "name": "Central Park is now something else",
+          "wikidata": "Q160409",
+          "osm:version": 1
+        },
+        "geometry": null
+      },
+      "oldVersion": null
+    },
+    {
+      "description": "Test OSM name matches with aliases on Wikidata",
+      "expectedResult": {},
```

Comment:
> shall we create a separate ticket for the same?

Yes please. We should have consistent design principles that will serve as a guide to build useful compare functions without being constrained by limitations of osmcha.